stock_location_orderpoint/__manifest__.py

```diff
@@ -24,6 +24,7 @@
         "demo/stock_location_orderpoint.xml",
     ],
     "depends": [
+        "stock_available_base_exclude_location",
```

Comment:
@jbaudoux As this adds a new dependency, I think a new module to extend is a good idea.